src/index.html

```diff
@@ -5,9 +5,53 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section class="card" data-qa="card">
+      <img
+      src="images/imac.jpeg"
+      alt="iMac"
+      class="card__img"
+      >
+      <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+      <div class="card__code">Product code: 195434</div>
+
+      <div class="card__grade">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+
+          <div class="stars__star"></div>
+        </div>
+
+        <div class="card__reviews">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-name">Price:</div>
+
+        <div class="card__price-cost">$2,199</div>
+      </div>
```

Comment:
let's use some semantic tags for text

Reply:
@etojeDenys  Hello, all other comments are clear, but this comment is not. I don't understand what tags can be used? These are not headings, span is also not suitable, others change the style of the text, but they are prohibited for use. Can you give me some advice?

Reply:
p tag will be good here, you can remove unnecessary default styles that bother you in semantic tags.